electron/js/menu/system.js

```diff
@@ -271,6 +271,7 @@ var linuxTemplate = {
   label: config.NAME,
   submenu: [
     localeTemplate,
```

Comment:
So before the languages add Preferences and separator.. here as well (without removing the preferences from Edit)